### ISSUE TITLE
Unclear browser support / suppression

### ISSUE THREAD

**Issue by arbelpeled** [OfficeDev/office-js-docs-pr#1907]
We're developing an add-in that cannot run on IE11, only Edge.

The text in the documentation regarding how to set a requirement so as to be sure the browser our add-in will run in will be Edge is really hard to follow.

IIUC, stating ExcelApi 1.9 and above would cause only Office 365 of a suitable version to be supported, which should give us Edge.
However, this is not really explicit, and also the validation tool for the manifest.xml file still states the add-in would be tested on Excel 2016 and higher (even though Excel 2016 is listed to only support ExcelApi 1.1).

Clarifications would be very welcome.
Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3c7080b5-9791-e1ca-ff77-ae3dbe5f10cc
* Version Independent ID: 1207aec0-6183-53bb-2768-ebb39d09ed99
* Content: [Excel JavaScript API requirement sets - Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/excel-api-requirement-sets)
* Content Source: [docs/reference/requirement-sets/excel-api-requirement-sets.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/reference/requirement-sets/excel-api-requirement-sets.md)
* Product: **excel**
* Technology: **add-ins**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by Rick-Kirkham** [comment]
The reason its not quite explicit is that we don't want developers to think in terms of supporting or not supporting _browsers_.  Occasionally, we do back port a new API to an old office, like Office 2016. Although, its extremely unlikely, we could someday back port recent ExcelApi Requirement Sets to Office 2016. You should think in terms of _requirement sets_ and let Office and Windows determine what browser is going to be used by the customer running your add-in, which you cannot control. (For details, see the article [Browsers used by Office Web Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/concepts/browsers-used-by-office-web-add-ins).)

The message you are getting about the testing is probably a bug. Since that is not an issue with the documentation, please raise it on Stack Overflow and tag your question with both `office-store` and `office-js`.
